index.html

```diff
@@ -69,124 +69,73 @@ <h1 class="main-title"> I'm Joaquin. <br>
-      <article class="card-works">
-        <img src="./img/Snapshoot Portfolio.png" alt="snapshootimg" class="snapshoot-img">
-        <img src="./img/desktop/Snapshoot1-desktop.svg" alt="snapshootdsk" class="snapshoot-img-dsk">
-        <div class="adjust">
-          <div class="leftblock">
-            <h2 class="card-works-title">Tonic</h2>
-            <div class="ref">
-              <span class="client">Canopy</span>
-              <span><i class="fa fa-circle fa-xs"></i></span>
-              <span>Back End Dev</span>
-              <span><i class="fa fa-circle"></i></span>
-              <span class="year">2015</span>
-            </div>
-            <p class="card-primary-text">A daily selection of privately personalized reads; no accounts or sign-ups
-              required.</p>
+    <section id="myModal" class="works-modal display-none">
+      <!-- The Modal -->
+      <div class="modal">
+        <!-- Modal content -->
+        <div class="frame-desktop">
+          <h2 class="frame-mobile-h2">Tonic</h2>
+          <span class="close-popup">&times;</span>
+        </div>
+        <div class="spans-popup">
+          <span class="client">Canopy</span>
+          <span><i class="fa fa-circle fa-xs"></i></span>
+          <span class="role"> Back End Dev</span>
+          <span><i class="fa fa-circle"></i></span>
+          <span class="year">2015</span>
+        </div>
+        <img src="./img/Snapshoot-Portfolio-popup.svg" alt="Snapshoot-Portfolio" class="img-popup" id="popupimage">
+        <img src="./img/snapshoot-portfolio-mobile-popup.svg" alt="mobile-popup" class="img-popup-mobile" id="popupimage">
+        <div class="modal-container"> 
+          <div class="information-popup">
+            <p class="paragraph-popup">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              <br>
+              <br>
+              <br>
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.
+            </p>
+            <p class="paragraph-popup-mobile">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+            </p>
```

Comment:
- [ ] Great work on creating the modal! :+1: However, as per the project requirements, it is necessary to create the modal using JavaScript, similar to how you did with the works section. This is because utilizing JavaScript will enable you to easily modify the information in each work, such as the title and image, which will help you ensure that the modal accurately reflects the information of each work.